akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -1008,8 +1008,7 @@ object TE extends ModeledCompanion[TE] {
 }
 final case class TE(acceptableEncodings: immutable.Seq[TransferEncoding]) extends jm.headers.TE with RequestHeader {
   def append(encodings: immutable.Seq[TransferEncoding]) = TE(this.acceptableEncodings ++ encodings)
-  import `Transfer-Encoding`.encodingsRenderer
-  def renderValue[R <: Rendering](r: R): r.type = r ~~ acceptableEncodings
+  def renderValue[R <: Rendering](r: R): r.type = r.~~(acceptableEncodings)(`Transfer-Encoding`.encodingsRenderer)
```

Comment:
Great if that fixes the compilation... I think we use the same pattern elsewhere as well but if we run into that now we know what to do ;).

Reply:
Yeah, for these simple kinds of problems it's perfectly fine, even a few characters saved... It was more problematic in this other test file which exercised the contrived FormField implicits where giving the params explicitly would have been quite a pain.